### ISSUE TITLE
Ship landing doesn't detect fuel.

### ISSUE THREAD

**Issue by PythonGamer51** [realthecaffiend/SOS2ExpSpinalEngines#19]
When attempting to land a ship using spinal engines I get the message: "Not enough fuel to move ship, required X fuel".

Taking off works fine.

I have not tested this with chem fuel exhaust, only nuclear. 

No messages how in the debug log when attempting to land.

Pleas let me know if there is any more information you need from me.

Edit: I just noticed the other issue that mentioned takeoff. I hadn't realized that this was an intentional choice. Would this just be a matter of editing the same XML file?

It would also be nice for this to be an option in the mod options. (I'm always worried about really messing something up no matter how simple the change is.)

**Comment by realthecaffiend** [comment]
Hey I see you already had seen the other issue: It's a little odd you were able to launch, did you just have other engines also attached at the time that provided enough fuel?

The problem I ran into in testing was kind of twofold with launching:
A) it encouraged building a giant ship on the surface to launch which feels like a player trap since you then skip most of the ships/content in space, and it takes forever unless you have other mods providing incredible plasteel etc, so it felt bad to encourage it. However, I didn't really think of subsequent landing and takeoff honestly (since I mostly play entirely in space in SOS2, with maybe a microlander for the endgame quests). A real demonstration of the importance of other player feedback here, I just didn't think of that!

B) liftoff in SOS2 is purely how much fuel you have, and I quickly found a ship with only one, maybe two, spinal engines that still had good T:W, the engines just didn't hold enough fuel to launch it so you'd have to launch a surprisingly small ship to even get it off the ground. Or build a bunch of smaller engines kind-of defeating the purpose? 

I like the option of some sort of a player option here though, I'm willing to ignore lots of design crimes in other games if I can just change them for my own game. Currently I'm in the middle of a big re-work to make the code less uh ugly frankly, and that would include the ability to have like "side attachments" on structures, not just in a single line. I was thinking of an option for larger fuel tanks to attach/some other structure (Uh, "gravity support", gimbals, something else haha) that would let them work as launch (and I'd have to test landing, of course). I think I'd like that more than a checkbox? I'm interested in thoughts on that, however!

**Comment by PythonGamer51** [comment]
Yeah sorry about the launch thing. It was like 1 am and I was super sleep deprived. I hadn't noticed that I had a nuclear engine on the ship. (it got destroyed on launch)

I usually start in space for my SOS2 runs so the original takeoff is almost completely ignored for me.

My current need for launching is for a very small ship I have that I use as a mobile base on the surface to collect anomalies to study up in orbit. I would gladly trade in some T:W for being able to lift off from the surface without having to use the default SOS2 engines since the destroy box is difficult to work around. (the spinal ones are slightly smaller)

Maybe a side addon that changes the "fuel mixture" to allow atmospheric use with a much higher consumption as a trade off?
